lib/customize.js

```diff
@@ -153,6 +153,12 @@ const Util = {
     Util.domModal.classList.remove("active");
   },
   saveFile : function saveFile(filename, mimeType, dataAsString) {
+    if (typeof Android !== "undefined" && typeof Android.saveFile === 'function') {
```

Comment:
Not clear to me why we have both customize.js and interface.js, so I updated both.